### PR TITLE
Set up version switcher and deploy first version (0.3.0)

### LINKED ISSUE #67: Versioned Docs
We'd like to introduce versioned docs since new releases may introduce breaking changes. Read the docs may be an option for hosting versioned docs for free using a tag on the docs to assign the version. @melissawm will explore if mkdocs is compatible with read the docs.

### PR DESCRIPTION
**Pull request by melissawm**
Please take a look at the `update_version.md` file for instructions on how to actually deploy a new version of the docs when a new Acquire release is done. Happy to address any other feedback.

Closes #67

### CODE REVIEW

File: versions.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "version": "0.3.0",
+    "title": "0.3.0",
+    "aliases": ["stable"]
+  },
+  {
+    "version": "dev",
+    "title": "dev",
+    "aliases": ["latest"]
+  }
+]
```
Comment (aliddell) [inline versions.json:12]: Will this need to be manually updated as well, or does mike autogenerate it?

File: docs/javascripts/version-select.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+window.addEventListener("DOMContentLoaded", function() {
+
+  // ABS_BASE_URL is the full URL
+  // e.g. https://acquire-project.github.io/acquire-docs/dev/get_started/
+  var ABS_BASE_URL = document.baseURI;
+  // This should probably not be hardcoded
+  var CURRENT_VERSION = ABS_BASE_URL.split("/")[4];
```
Comment (aliddell) [inline docs/javascripts/version-select.js:7]: Can we do a regex search for a semver string (or "dev") instead? I agree, I don't like that this is hardcoded. `(\d+\.\d+\.\d+(\-?rc\d+)?|dev)` might do the trick:
image

File: versions.json (hunk 1 above)
Comment (melissawm) [reply versions.json:12]: This is created and updated by `mike`, but we can also manually edit it if necessary.

**Comment by melissawm** [comment]
This should finally be ready 😄 Sorry for the delay. I tried explaining the release process as well but if we want to automate that from the acquire-python repo it should be possible as well.

**Comment by melissawm** [comment]
This should be ready now :)

**Review by aliddell — APPROVED** [review]

**Pull request merged**